tests/keras/backend/backend_test.py

```diff
@@ -159,6 +167,30 @@ def check_two_tensor_operation(function_name,
     assert_list_keras_shape(t_list, z_list)
 
 
+def check_three_tensor_operation(function_name,
+                                 x_shape_or_val,
+                                 y_shape_or_val,
+                                 z_shape_or_val,
+                                 backend_list,
+                                 **kwargs):
+    x_shape, x_val = parse_shape_or_val(x_shape_or_val, "where")
+    y_shape, y_val = parse_shape_or_val(y_shape_or_val)
+    z_shape, z_val = parse_shape_or_val(z_shape_or_val)
+
+    t_list = []
+    z_list = []
+    for k in backend_list:
+        if k != KC:
```

Comment:
`check_three_tensor_operation` is supposed to be generic. If you want to exclude cntk, you should do it before calling this function.

Reply:
ok, i have it fixed.